app/scripts/metamask-controller.test.js

```diff
@@ -455,6 +480,21 @@ describe('MetaMaskController', () => {
         addresses.forEach((address) => {
           expect(identities).toContain(address);
         });
+
+        const internalAccounts =
+          metamaskController.accountsController.listAccounts();
+
+        internalAccounts.forEach((account) => {
+          expect(addresses.includes(account.address) === true).toBe(true);
```

Comment:
Nit: The `=== true` is implied:

``` suggestion
          expect(addresses.includes(account.address)).toBe(true);
```

You can also say:

``` suggestion
          expect(addresses).toContain(account.address);

Reply:
Solved in [36c3984](https://github.com/MetaMask/metamask-extension/pull/21437/commits/36c398471c2e87473348764ce2085f917d592085)